File: sumarhastaelmayordedos.py
import psycopg2
import math
from tabulate import tabulate
def  post(x,y,z):
    try:
        conexion = psycopg2.connect(
            host = "localhost",
            port = "5432",
            user = "postgres",
            password = "jblue5queso",
            dbname = "tarea1"
            )
    except:
        print("Sin Conexion Exitosa\n")
            
    cursor = conexion.cursor()
    cursor.execute("INSERT INTO ordenarmayamen(pn, sn, res) VALUES (%s, %s, %s);", (x, y, z))
    conexion.commit()
    cursor.close()
    conexion.close()
mat=[]
a = int(input("Ingrese el primer numero: "))
b = int(input("Ingrese el segundo numero:  "))
if a>b:
    for y in range(a,b-1,-1):
        mat.append(y)
if b>a:
    for y in range(b,a-1,-1):
        mat.append(y)

print("La sucesion es: \n ",mat)
post(a,b,mat)

File: sumarhastaelmayordedos.m
pkg load database;
clc; 
clear all; 
close all;
pri = input('Ingrese el primer numero: ');
seg = input('Ingrese el segundo numero: ');
sum=1;
params= cell(1,3);
if pri>seg
    params{1,1}=pri;
    params{1,2}=seg;
  for x=pri:-1:seg
    V(sum)=x;
    sum=sum+1;
  endfor
endif

if seg>pri
    params{1,1}=seg;
    params{1,2}=pri;
  for x=seg:-1:pri
    V(sum)=x;
    sum=sum+1;
    
  endfor
endif

disp(V)

%Conectar con PGADMIN
linearecta=mat2str(V);
params{1,3}=linearecta;

conn = pq_connect(setdbopts('dbname','tarea1','host','localhost',
'port','5432','user','postgres','password','jblue5queso'));
N=pq_exec_params(conn, "insert into ordenarmayamen(pn,sn,res) values($1,$2,$3);",params); %insertar datos en la tabla

%Texto
save('sumarhastaelmayordedos.txt','V','-ascii');